fe1-web/src/features/evoting/screens/__tests__/CreateElection.test.tsx

```diff
@@ -45,10 +72,96 @@ describe('CreateElection', () => {
 
   it('renders correctly when name and question are empty', () => {
     const component = render(
-      <FeatureContext.Provider value={contextValue}>
-        <MockNavigator component={CreateElection} />
-      </FeatureContext.Provider>,
+      <Provider store={mockStore}>
+        <FeatureContext.Provider value={contextValue}>
+          <MockNavigator component={CreateElection} />
+        </FeatureContext.Provider>
+      </Provider>,
     ).toJSON();
     expect(component).toMatchSnapshot();
   });
+
+  it('renders correctly when name and questions are whitespace string', () => {
+    const { getByTestId, toJSON } = render(
+      <Provider store={mockStore}>
+        <FeatureContext.Provider value={contextValue}>
+          <MockNavigator component={CreateElection} />
+        </FeatureContext.Provider>
+      </Provider>,
+    );
+    const electionName = getByTestId('election_name_selector');
+    const question = getByTestId('question_selector_0');
+    fireEvent.changeText(electionName, '     ');
+    fireEvent.changeText(question, '        ');
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('sends correctly data on edited input that should be trimmed', () => {
+    const badQuestions: QuestionState[] = [
+      {
+        question: 'Question 0',
+        ballot_options: ['Answer 0', 'Answer 1'],
+        voting_method: STRINGS.election_method_Plurality,
+        id: '',
+        write_in: false,
+      },
+      {
+        question: '   Question     1        ',
+        ballot_options: ['    Answer 0    ', '  Answer 1  ', 'will be removed'],
+        voting_method: STRINGS.election_method_Plurality,
+        id: '',
+        write_in: false,
+      },
+    ];
+    jest.setSystemTime(Date.UTC(2023, 3, 21));
```

Comment:
Why is this necessary? If you want to workaround specific time values you can always use https://jestjs.io/docs/expect#expectanything

Reply:
It is required, because I want the ID to be checked too. And it depends on the current time.

Reply:
I see 👍
I guess you did not come across this yet but the global system time should already be fixed for all tests here: https://github.com/dedis/popstellar/blob/master/fe1-web/jest/setupFakeTimers.js#L6 :)